tests/cache-cmd.earth

```diff
@@ -0,0 +1,20 @@
+VERSION 0.6

```

Comment:
Some users might want to have two (or more caches) within a target. It would be good to validate that works.

Another use-case could be defining the cache at the top of the Earthfile (under the implicit `+base` target). For example, we might need something along the lines of:

```
VERSION 0.6
CACHE /non-persistant-cache
CACHE --persists /persistant-cache

foo:
    RUN echo data >> /non-persistant-cache/data.txt
    RUN echo data >> /persistant-cache/data.txt
    RUN touch /fake-dep
    SAVE ARTIFACT /fake-dep

bar:
    RUN test $(cat /non-persistant-cache/data.txt | wc -l) -gt 0
    RUN test $(cat /persistant-cache/data.txt | wc -l) -gt 0
    SAVE IMAGE myimage:test

test:
    COPY +foo/fake-dep .
    IF [ -z "$(cat fake-dep)" ]
        BUILD +bar
    ELSE
        RUN false # should never happen
    END
```

Then on top of that, we should test the persisted cache made it to `myimage:test`:

```
docker run --rm myimage:test cat /persistant-cache/data.txt | grep 'data'
# and a similar one to make sure the non-persitant cache doesn't exist.
```

Reply:
Apparently it's not easy to save an image in our tests, as the operation fails with `io: read/write on closed pipe`. The other tests that utilize SAVE_IMAGE seem to be using `--no-output` (so they don't actually write the image) or testing something with pushing to a registry